opcua/common/xmlexporter.py

```diff
@@ -155,6 +155,12 @@ def add_etree_object_type(self, node):
 
     def add_variable_common(self, node, el):
         dtype = node.get_data_type()
+
+        # FIXME hack because get_data_type() has issues
+        if dtype.NamespaceIndex > 50:
```

Comment:
Use `parent.add_method()` and step through the code. Eventually `_create_method()` is called in manage_nodes.py where the InputArgument and OutputArgument properties are automatically created. When the properties are created the Identifier ends up in the `ns` field.